gradle/mod.gradle

```diff
@@ -269,9 +269,9 @@ def versionNumber = opc.hasPropertySet("modVersion") ? opc.getProperty("modVersi
 def versionSuffix = opc.hasPropertySet("modVersionSuffix") ? opc.getProperty("modVersionSuffix") : "alpha"
 def versionString
 if (versionSuffix == "") {
-    versionString = versionNumber + "-" + project.exactMinecraftVersion
+    versionString = project.exactMinecraftVersion + "-" + versionNumber
 } else {
-    versionString = versionNumber + "-" + project.exactMinecraftVersion + versionSuffix
+    versionString = project.exactMinecraftVersion + "-" + versionNumber + versionSuffix
```

Comment:
This shoudl be controllable with a version flag, since some of our mods already published using this format and now switching it back is not a good idea.

Reply:
It already was the other way around, but I messed this up during porting back from ng7

Reply:
Should not matter, I already publised two projects using this format, and I would rather not go back. So please add a toggle, that can be set to the default format that you are configuring it now to in this PR, however I would like to be able to switch to the format that you are now replacing in those already published projects.